### ISSUE TITLE
API Keys

### ISSUE THREAD

**Issue by FiveEightyEight** [FiveEightyEight/Noice_Tube#75]
Have back up API keys for failures

**Comment by JorgeBillini** [comment]
Detect code 503 in http status -> retry logic with new API_KEY

**Comment by FiveEightyEight** [comment]
Is this an error you're currently getting?
Unless you've been using the API key we got from Google it more than likely expired and we'll have to get new ones.

**Comment by JorgeBillini** [comment]
Need em keys

**Comment by JorgeBillini** [comment]
Is it possible to use ENV variables in a React Environment and if so, does netlify allow us to produce multiple env variables?

**Comment by FiveEightyEight** [comment]
Environment variables are a _thing_ in front ends React apps. I'm not sure about Netlify having environment variables but according to this post `https://stackoverflow.com/questions/48453493/set-environment-variable-for-build-in-netlify` you can create your own during the build process. 

Do you want environment variables to mask the API keys?

**Comment by JorgeBillini** [comment]
Yes. that way we can keep a collection of API_KEYS and each time one fails we can swap it out for another one. Wondering about the env variables since I have not had to use them for my React Projects
